### ISSUE TITLE
[BUG]   I can't install Flash Firmware to Nextion Screen

### ISSUE THREAD

**Issue by patryk0511** [joBr99/nspanel-lovelace-ui#606]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._

Flashing Firmware to Nextion Screen fails with an error : "Something has gone wrong flashing display firmware"

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [ ] Provide your apps.yaml (or your config):
```
  apps.yaml here:


```
- [ ] Go to Settings>Add-ons>AppDaemon>Log (or the output of your TS script) and then, provide the output of the log during your issue/bug occurs:
```
  Log output here:

Tasmota console

22:52:52.220 CMD: FlashNextion http://nspanel.pky.eu/lui-release.tft
22:52:52.232 MQT: stat/tasmota_0BF980/RESULT = {"FlashNextion":"Done"}
22:52:52.270 FLH: host: nspanel.pky.eu, port: 80, get: /lui-release.tft
22:52:52.834 FLH: Send (High Speed) flash 
22:53:08.334 FLH: Something has gone wrong flashing display firmware [bytes('8080808080008080808000800000008080800000000000808080008080000080...')]

```

### TO REPRODUCE
_Steps to reproduce the behavior:_
When i paste the script in the tasmota console: FlashNextion http://nspanel.pky.eu/lui-release.tft

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._
I can't go through the installation process, no progress bar appears on the Ns panel screen within 10 minutes.

### SCREENSHOTS
_If applicable, add screenshots/pictures to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._
_Please note here in case you are using ioBroker_

### PANEL / FIRMWARE VERION
_Please add the Panel/Firmware Version you are using (EU, US-L or US-P)_
NSPanel EU,  Tasmota 12.0.2 by Theo Arends

**Comment by joBr99** [comment]
Hey are you using the latest berry driver? (published 3 days ago)

Which tasmota version do you have is it really 12.0.2 or 12.0.2.X?
![image](https://user-images.githubusercontent.com/29555657/205402266-85ed75b4-ada4-4bd6-8ddb-0de012a04e3e.png)

**Comment by patryk0511** [comment]
sorry, it's all working now I looked at:   #601 [BUG] Flashing Issues on Tasmota Dev Version after 12.2.0.3 #601.
In this moment i downgrade Tasmota to 12.2.0 and flashing

**Comment by joBr99** [comment]
This bug is solved and it should also work with 12.2.0.4 and later

**Comment by patryk0511** [comment]
In my case it doesn't work so i downgraded to 12.2.0.4
